test/pummel/test-hash-seed.js

```diff
@@ -1,20 +1,21 @@
 'use strict';
 
+// Check that spawn child doesn't create duplicated entries
+require('common');
```

Comment:
This does not seem to work without relative paths, can you change it to `require('../common')`?